README.md

```diff
@@ -1,76 +1,20 @@
 open-budget-search-api
 ======================
 
-Install:
+### Setup environment and load data
 
-	Elasticsearch part
+* Follow the README from [this branch of budgetkey data pipelines](https://github.com/OriHoch/budgetkey-data-pipelines/tree/load_data_take3) README to setup the docker-compose and load data to ES
+* `docker build -t budgetkey/open-budget-search-api .`
+* If you used the default docker compose environment from budgetkey-data-pipelines, this should work:
+  * `docker run -d --name open-budget-search-api --rm -p18000:8000 --network budgetkeydatapipelines_default -eES_HOST=elasticsearch -eES_PORT=9200 budgetkey/open-budget-search-api`
+* app should be available at localhost:18000
+  * http://localhost:18000/search/exemptions/web/2000-01-01/2019-01-01/4/0
+  * http://localhost:18000/search/budget,national-budget-changes,contract-spending,entities/%D7%9E%D7%99%D7%A7/1992-01-01/2019-01-01/10/0
 
-		On windows:
+### Development
 
-		1. Download elasticsearch 1.7.0 from:
-		https://www.elastic.co/downloads/past-releases/elasticsearch-1-7-0
-		2. Go to the bin direcotry and execture:
-		plugin --install analysis-hebrew --url https://bintray.com/artifact/download/synhershko/elasticsearch-analysis-hebrew/elasticsearch-analysis-hebrew-1.7.zip
-		3. download all the hebrew data files from:
-		https://github.com/synhershko/HebMorph/tree/master/hspell-data-files
+* Run main.py -  This module start listening to requests of the following structure:
+  http://localhost:5000/search/comma_seperated_table_names/search_term/from_data/to_date/maxinum_size_of_result/offset
+  for example - http://localhost:5000/search/exemption/test/2000-01-01/2019-01-01/4/0
+* There is a log for exceptions called obudget.log which is written right next to the main.py
```

Comment:
which ES instance will it connect to by default?
Can you also add the relevant docker commands for this case?